modelexpress_client/python/modelexpress/load_strategy/trainer_pull_strategy.py

```
@@ -0,0 +1,234 @@
+        except Exception as e:
+            logger.debug("[Worker %d] GetTrainerTable RPC failed: %s", ctx.global_rank, e)
+
+        # Redis fallback
+        redis_url = envs.MX_REDIS_URL
+        try:
+            import redis as redis_lib
+            r = redis_lib.from_url(redis_url)
+            return r.get(key)
+        except ImportError:
+            logger.debug("[Worker %d] redis-py not installed", ctx.global_rank)
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)